### PR TITLE
Consistent batch size

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

**Comment by parkererickson-tg** [comment]
DO NOT MERGE: running tests first, then expanding to all other loaders.

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: SUCCESS, e2e Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/688/

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: SUCCESS, e2e Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/690/

File: pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql [hunk 1]
```diff
@@ -72,20 +79,40 @@ CREATE QUERY hgt_loader_{QUERYSUFFIX}(
         # Get seeds
         IF input_vertices.size()==0 THEN
             start = {seed_types};
+            HeapAccum<ID_Tuple> (1, tmp_id ASC) @@batch_heap;
+            @@batch_heap.resize(batch_s);
```
Comment (billshitg) [inline pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql:83]: Is the resize() here due to that capacity in the heap definition cannot be a variable?

File: pyTigerGraph/gds/gsql/dataloaders/neighbor_loader.gsql [hunk 2]
```diff
@@ -64,7 +66,11 @@ CREATE QUERY neighbor_loader_{QUERYSUFFIX}(
                 POST-ACCUM s.@tmp_id = getvid(s);
         END;
     END;
-
+    IF batch_size IS NULL THEN
+      batch_s = ceil(res.size()/(num_batches-1));
```
Comment (billshitg) [inline pyTigerGraph/gds/gsql/dataloaders/neighbor_loader.gsql:70]: same here

File: pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql [hunk 3]
```diff
@@ -72,20 +79,40 @@ CREATE QUERY hgt_loader_{QUERYSUFFIX}(
...
             IF filter_by IS NOT NULL THEN
+                FOREACH iter IN RANGE[0,1-1] DO
```
Comment (billshitg) [inline pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql:85]: Is this number 1 (and 10 below) a placeholder?

File: pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql (hunk 1 above)
Comment (parkererickson-tg) [reply pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql:83]: Unfortunately that is the case.

File: pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql (hunk 3 above)
Comment (parkererickson-tg) [reply pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql:85]: I caught this as well as I was converting other queries. It should be the variable corresponding to the number of heap inserts, to be parameterized.

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/697/

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/698/

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/699/

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/700/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

**Review by billshitg — APPROVED** [review]

**Pull request merged**